docs/Datastore/Connection.md

```diff
@@ -0,0 +1,106 @@
+# High-level Datastore connection
+
+`RapidCore.GoogleCloud.Datastore.DatastoreConnection` is a high-level connection class providing basic methods for working with Entities using the [the ORM](./Orm).
+
+It is not meant to cover every single use-case directly. For the more exotic use-cases, it provides direct access to the low-level connection from Google's SDK.
+
+
+## Creating an instance
+
+```csharp
+using RapidCore.GoogleCloud.Datastore;
+
+var datastoreDb = Google.Cloud.Datastore.V1.DatastoreDb.Create("project-id", "namespace");
+var connection = new DatastoreConnection(datastoreDb);
+```
+
+
+
+## Inserting an entity
+
+```csharp
+public class KewlPoco
+{
+    public string Id { get; set; }
+    public string Oh { get; set; }
+}
+
+// instantiate the poco
+var poco = new KewlPoco { Id = "hot-damn", Oh = "yeah" };
+
+// insert it
+await connection.InsertAsync<KewlPoco>(poco);
+```
+
+
+## Updating an entity
+
+```csharp
+public class KewlPoco
+{
+    public string Id { get; set; }
+    public string Oh { get; set; }
+    public int IntOfDoom { get; set; }
+}
+
+// load the document
+var poco = ...load entity...;
```

Comment:
I think we should provide a real example here on how to load the code. We must remember that the users coming in to the documentation are probably new to the ORM..

Reply:
As I recall the next example on that page is actually a "load an entity"-example.